_tuning-your-cluster/availability-and-recovery/remote-store/index.md

```diff
@@ -17,202 +17,80 @@ Introduced 2.10
 
-## The `index.translog.durability` translog setting
-
-Without remote-backed storage, indexing operations are only persisted to disk when the translog is fsynced. Therefore, any data that has not been written to disk can potentially be lost. 
-
-The `index.translog.durability` setting controls how frequently OpenSearch fsyncs the translog to disk:
-
-- By default, `index.translog.durability` is set to `request`. This means that fsync happens after every request, and all acknowledged write requests persist in case of failure.
-
-- If you set `index.translog.durability` to `async`, fsync happens periodically at the specified `sync_interval` (5 seconds by default). The fsync operation is asynchronous, so acknowledge is sent without waiting for fsync. Consequently, all acknowledged writes since the last commit are lost in case of failure.
-
-With remote-backed storage, the translog is uploaded to a remote store for durability.
-
-`index.translog.durability` is a dynamic setting. To update it, use the following query:
-
-```json
-PUT my_index/_settings
-{
-  "index" : {
-    "translog.durability" : "request"
-  }
-}
-```
-
-## Refresh-level and request-level durability
-
-The remote store feature supports two levels of durability:
-
-- Refresh-level durability: Segment files are uploaded to remote store after every refresh. Set the `remote_store` flag to `true` to achieve refresh-level durability. Commit-level durability is inherent, and uploads are asynchronous.
-
-  If you need to refresh an index manually, you can use the `_refresh` API. For example, to refresh the `my_index` index, use the following request:
-   
-  ```json
-  POST my_index/_refresh
-  ```
+After segments are created on the primary shard as part of the refresh, flush, and merge flow, the segments are uploaded to remote segment store and the replica shards source a copy from the same remote segment store. This frees up the primary shard from having to perform a data copying operation.
 
-- Request-level durability: Translogs are uploaded before acknowledging the request. Set the `translog` flag to `true` to achieve request-level durability. In this scenario, we recommend to batch as many requests as possible in a bulk request. Batching requests will improve indexing throughput and latency compared to sending individual write requests.
+## Configuring remote-backed storage
 
-## Enable the feature flag
+Remote-backed storage is a cluster level setting. It can only be enabled when bootstrapping to the cluster. After bootstrapping completes, the remote-backed storage cannot be enabled or disabled. This provides durability at the cluster level.
```

Comment:
```suggestion
Remote-backed storage is a cluster-level setting. It can only be enabled when bootstrapping to the cluster. After bootstrapping completes, the remote-backed storage cannot be enabled or disabled. This provides durability at the cluster level.
```